### PR TITLE
auto OCP-42925

### PR DESCRIPTION
**Pull request by huiran0826**
Auto OCP-42925
Test log: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/3687/console

/cc @openshift/team-sdn-qe

### CODE REVIEW

File: features/networking/ovn-egress-ip.feature [hunk 1]
```diff
@@ -870,3 +870,46 @@ Feature: OVN Egress IP related features
     Then the step should succeed 
     And the output should not contain:
       | name=egressip |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-42925
+  @admin
+  @destructive
+  @4.10 @4.9
+  @vsphere-ipi
```
Comment (zhaozhanqi) [inline features/networking/ovn-egress-ip.feature:879]: maybe need `@network-ovnkubernetes` also for CI

Comment (huiran0826) [reply features/networking/ovn-egress-ip.feature:879]: If add @network-ovnkubernetes, will it be selected by other platforms?

Comment (zhaozhanqi) [reply features/networking/ovn-egress-ip.feature:879]: this should be for new Prow CI
@liangxia could you help explain more for the tags added recently?

Comment (liangxia) [reply features/networking/ovn-egress-ip.feature:879]: You can refer to https://github.com/openshift/verification-tests/blob/master/doc/tags.adoc

Comment (huiran0826) [reply features/networking/ovn-egress-ip.feature:880]: Thanks for the info.

**Pull request merged**